### PR TITLE
Groups can have multiple organizers

### PR DESCRIPTION
**Pull request by Raemon**

### CODE REVIEW

**Comment by Raemon** [comment]
Note: this currently has some problems. You apparently can't remove people from the organizer's list (ever)

**Comment by Raemon** [comment]
(I think this PR is essentially fine, and the issue is that our UsersListSearch component doesn't let you remove Users, which I think should be it's own PR)

File: packages/lesswrong/components/form-components/UsersListEditor.jsx [hunk 1]
```diff
@@ -109,23 +106,21 @@ class UsersListEditor extends Component {
...
-            />
-          </Components.ErrorBoundary>
-          <SortableList
-            axis="xy"
-            items={this.state.userIds}
-            onSortEnd={this.onSortEnd}
-            currentUser={currentUser}
-            removeItem={this.removeUserId}
-            shouldCancelStart={this.shouldCancelStart}
+      <div className={classes.search}>
```
Comment (Discordius) [inline packages/lesswrong/components/form-components/UsersListEditor.jsx]: General rule for Material-UI is that the root level element has `classes.root` applied to it. So I think we should stick to that.

**Comment by Discordius** [comment]
Rest seems fine.

**Comment by Raemon** [comment]
Updated to use root

**Comment by Discordius** [comment]
Where are the circleCI tests?

**Comment by Discordius** [comment]
It seems that you canceled the circleCI tests for this one somehow. Or maybe someone else did. In either case, I restarted them.

**Pull request merged**